### ISSUE TITLE
FeatureRequest: Add option for anonymizeIp

### ISSUE THREAD

**Issue by jhinzmann** [neos/neos-googleanalytics#25]
If you use this plugin and want to set the anonymizeIp option, you have to extend the `TYPO3.Neos.GoogleAnalytics:TrackingCode` prototype.
Since Neos is mostly used in areas where the anonymizeIp option is mandatory I guess it could be added as option for easier integration?

**Comment by rolandschuetz** [comment]
I agree. Would be nice to have a checkbox for that in the Administration Area. 

For everyone who needs this now:
```
prototype(Neos.GoogleAnalytics:TrackingCode) {
	code.@process.anonymizeIp = ${"ga('set', 'anonymizeIp', true);" + value}
}
```

**Comment by kitsunet** [comment]
I probably wouldn't want to set that via backend, but some easier way to configure that and maybe the anonymize default would make sense.

**Comment by kdambekalns** [comment]
With PR #36 merged, this is a simple Fusion change now…
